src/shared/network.ts

```diff
@@ -1,12 +1,30 @@
 import { Networking } from "@flamework/networking";
 
-interface ServerEvents {}
+/*
 
-interface ClientEvents {}
+	Server Recievers
+	Network messages that can be sent from clients and recieved from the server
+
+*/
+interface ServerEvents {}
 
 interface ServerFunctions {}
 
+/*
+
+	Client Recievers
+	Network messages that will be sent from the server and recieved by client(s)
+
+*/
+interface ClientEvents {}
+
 interface ClientFunctions {}
 
+/*
+
+	Initialization
+	Creates and links the networking Remotes in ReplicatedStorage.
+
+*/
```

Comment:
I feel this comment is unnecessary. Doc comments on the individual variable declarations might make sense but I'd personally leave them out in this case.